db/seeds/01_alta.rb

```diff
@@ -117,41 +117,55 @@
     zipcode: "12345"
   )
 
+  user_profile_params = ->(adopter_foster_account) {
```

Comment:
I actually prefer this approach because it saves us from repeating the profile attributes over and over again. But, it is different from how it's being handled above. So, could you please create an issue to refactor the seed files to use this approach for all cases where we create an adopter foster profile? That would help clean the seeds up nicely!